src/airstack/execute_query.py

```diff
@@ -231,7 +231,7 @@ async def get_prev_page(self, query, variables, page_info):
             document_ast = parse(next_query)
             if has_cursor(document_ast, _page_info_key):
                 replace_cursor_value(document_ast, _page_info_key,
-                _page_info_value['prevCursor'])
+                _page_info_value['prevCursor'], self.variables)
```

Comment:
Should we not throw the error if `_page_info_value['prevCursor']` is an empty string?